packages/parser/src/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { JSXElement, createMemo } from "solid-js";
+
+export function createParser(id?: string) {
+  const $TOKEN = Symbol(id || "solid-parser");
+
+  function tokenize<
+    TToken extends Meta & {
+      props: unknown;
+    }
+  >(meta: Omit<TToken, "props">): (props: TToken["props"]) => JSXElement {
+    return (props: any) => {
+      return Object.assign(
+        () => {
+          console.info("tokens can only be rendered inside a Parser");
+          return <></>;
```

Comment:
Can just return an empty string. No need to make it a tsx file and compile it.
```suggestion
          return '';
```